sway/config.c

```diff
@@ -520,8 +522,13 @@ bool load_main_config(const char *file, bool is_active, bool validating) {
 		return success;
 	}
 
-	if (is_active) {
+	if (is_active && !validating) {
 		input_manager_verify_fallback_seat();
+
+		for (int i = 0; i < config->input_configs->length; i++) {
+			input_manager_apply_input_config(config->input_configs->items[i]);
```

Comment:
This call seems to have been added. Is there a reason why we didn't need it and need it now?

Reply:
It's not being handled in `cmd_input` on reload anymore

https://github.com/swaywm/sway/blob/33be52b26a2b83add6be469de8840ecb42f3002c/sway/commands/input.c#L92-L94

Reply:
Nevermind, the old call now has a `if (config->reloading)` guard